rat/rat_left/female/iqm.cfg

```diff
@@ -1,15 +1,17 @@
-//  rat_left for female   - by Dziq
+//  animated rat_left for female   - by Dziq
 //      ....
-        iqmload rat_left_female.iqm
+		iqmload rat.iqm
```

Comment:
Use of the TAB character is not permitted, indentation in Red Eclipse must be four spaces.